transformer/transforms/util/import_csv.py

```diff
@@ -0,0 +1,92 @@
+from transformer.registry import register
+from transformer.transforms.base import BaseTransform
+import csv
+import urllib
+import tempfile
+
+#Approximate size of a 1000 line CSV with 14 columns, some tezt in each
+#This takes ~3-5 seconds to turn into line-items in Zapier, and is still workable in the editor
+#So not arbitrary, but I'm sure it could be tweaked
+MAX_CSV_FILE_SIZE = 150000
+
+class UtilImportCSVTransform(BaseTransform):
+
+    category = "util"
+    name = "import_csv"
+    label = "Import CSV File"
+    help_text = (
+        "Import a CSV file from a public URL, File field from another Zap step, or entered text.  "
+        "Limited to 150k (around 1000 rows).  "
+        "Output is a Line-item field for each column, and a text field with CSV file contents.  "
+        "More on using csv files [here.](https://zapier.com/help/formatter/#how-process-csvs-formatter)"
+    )
+
+    noun = "CSV"
+    verb = "import"
+
+    def build_input_field(self):
+        return {
+            "type": "file",
+            "required": True,
+            "key": "inputs",
+            "label": "CSV file",
+        }
+
+    def transform(self, csv_url, **kwargs):
+        # Take a file input and output a set of line-item fields, or a big string field
+
+        if not csv_url:
+            return u''
+
+        # create a temp file, then load the csv into it
+        response = tempfile.NamedTemporaryFile()
+        response.seek(0)
+        urllib.urlretrieve(csv_url, response.name)
+
+        #check file size
+        response.seek(0, 2)
+        size = response.tell()
+        if (size > MAX_CSV_FILE_SIZE):
+            self.raise_exception('CSV Import only supports file sizes < 150K.')
+
+        # use csv utils to see if there is a dialect, if the file is malformed in anyway, this will fail
+        response.seek(0)
+        #try:
+        dialect = csv.Sniffer().sniff(response.read())
+        #except:
+            #self.raise_exception('This CSV file has a format that Formatter does not understand')
+        # worked, so let's look for a header
+        response.seek(0)
+        header = csv.Sniffer().has_header(response.read())
+
+        response.seek(0)
+
+        output = {"Line-item(s)": [],"CSV text": "", "Header": header}
```

Comment:
Can you change these dict properties to be more pythonic? this might look like:
```suggestion
        output = { "line_items": [], "csv_text": None, "header": header }
```